src/HealthcareApis/HealthcareApis/HealthcareApis/NewAzureRmHealthcareApisService.cs

```diff
@@ -277,6 +283,16 @@ private Kind ParseKindFromVersion(string fhirVersion)
             return CosmosOfferThroughput;
         }
 
+        private string GetCosmosDBKeyVaultKeyUri()
+        {
+            if (CosmosKeyVaultKeyUri == null)
```

Comment:
I think the logic in this method works fine, but might be unnecessary, as the default KeyVaultKeyUri value is null.  So, in theory, this method could just do

return CosmosKeyVaultKeyUri;

Reply:
I agree, was just making it consistent with the other code. Will update

Reply:
Yes, your goal of consistency makes sense.  Arguably, maybe the consistency you gain with this is work the extra code. I can go either way here.  I will leave it to you -- feel free to leave it as is if you prefer it this way.